src/platforms/node/common/typescript.mdx

```diff
@@ -2,6 +2,7 @@
 toc: true
 title: TypeScript
 sidebar_order: 500
+description: "Learn more about using your Sentry SDK and SourceMaps with TypeScript."
```

Comment:
Source Maps*, it should be written as 2 separate words.

Reply:
thanks.